ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml

```diff
@@ -359,7 +370,7 @@ StatusSectionLayout {
         id: d
 
         property int currentIndex: 0
-        property int mintTokensSettingsIndex
+        property int mintTokensSettingsIndex: 3 // TODO: Get stackView index corresponding to the mintPanel
```

Comment:
Why not do this properly here and now?

Reply:
Already solved in this commit: https://github.com/status-im/status-desktop/commit/23f493f251db429d9accea9c805af58f7a42ba42